locale/sv/LC_MESSAGES/django.po

```diff
@@ -743,151 +743,204 @@ msgstr ""
 "Observera att deltagarens och deltagarens kontaktuppgifter kan vara "
 "information från olika personer."
 
-#: registrations/filters.py:77
+#: registrations/filters.py:85
 msgid "Invalid registration ID(s) given."
 msgstr ""
 
-#: registrations/models.py:50 registrations/models.py:655
+#: registrations/models.py:52 registrations/models.py:756
 msgid "City"
 msgstr "Stad"
 
-#: registrations/models.py:51 registrations/models.py:634
-#: registrations/models.py:793
+#: registrations/models.py:53 registrations/models.py:735
+#: registrations/models.py:914
 msgid "First name"
 msgstr "Förnamn"
 
-#: registrations/models.py:52 registrations/models.py:641
-#: registrations/models.py:800
+#: registrations/models.py:54 registrations/models.py:742
+#: registrations/models.py:921
 msgid "Last name"
 msgstr "Efternamn"
 
-#: registrations/models.py:55 registrations/models.py:676
+#: registrations/models.py:57 registrations/models.py:777
 msgid "ZIP code"
 msgstr "Postnummer"
 
-#: registrations/models.py:87
+#: registrations/models.py:90
+msgid "You must provide either signup_group or signup."
+msgstr ""
+
+#: registrations/models.py:94
+msgid "You can only provide signup_group or signup, not both."
+msgstr ""
+
+#: registrations/models.py:102
 msgid "Created at"
 msgstr "Skapad kl"
 
-#: registrations/models.py:93
+#: registrations/models.py:108
 msgid "Modified at"
 msgstr "Ändrad kl"
 
-#: registrations/models.py:196
+#: registrations/models.py:211
 msgid "You may not delete a default price group."
 msgstr ""
 
-#: registrations/models.py:204
+#: registrations/models.py:219
 msgid "You may not edit a default price group."
 msgstr ""
 
-#: registrations/models.py:210
+#: registrations/models.py:225
 msgid ""
 "You may not change the publisher of a price group that has been used in "
 "registrations."
-msgstr "Du får inte ändra utgivare för en prisgrupp som har använts vid "
+msgstr ""
+"Du får inte byta utgivare för en prisgrupp som har använts vid "
```

Comment:
Should this be "Du får inte byta utgivare för en kundgrupp som har använts vid "
"registreringar."

Reply:
This should also use the customer group term -> will fix soon